src/printer.js

```diff
@@ -4125,11 +4126,14 @@ function isMemberExpressionChain(node) {
 // type T = { method: () => void };
 // type T = { method(): void };
 function isObjectTypePropertyAFunction(node) {
+  // This is fragile, as it assumes there's only one space between 'get'/'set'
+  // and the method name.
+  const kindOffset = node.kind === 'get' || node.kind === 'set' ? 4 : 0;
```

Comment:
Hmm, this is actually worse than I thought, as putting multiple spaces after 'get' formats to invalid code, which wasn't the case before this change:

```bash
$ prettier --parser flow <(echo 'type T = { get   method(): void}') --debug-check
/dev/fd/63
/dev/fd/63: SyntaxError: Unexpected token => (1:24)
> 1 | type T = { get method() => void };
    |                         ^
  2 |
```

Maybe I can have it fall back to the old behavior if there are multiple spaces. It's not ideal, but at least it would fix the AST in the most common case.